src/globalInfo/types.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { JsonMap, Optional } from '@salesforce/ts-types';
+
+export enum SfInfoKeys {
+  ORGS = 'orgs',
+  TOKENS = 'tokens',
+  ALIASES = 'aliases',
+}
+
+export type Timestamp = { timestamp: string };
```

Comment:
@mdonnalley what is the relationship between Timestamp type and the timestamp property in a structured config value? Here is is defined as a string, but in [GlobalInfoConfig](https://www.npmjs.com/package/@salesforce/plugin-analytics) it is a Date.

Reply:
The Date gets converted to a string when it gets written to the file. I think @RodEsp had some ideas around how to improve the types here

Reply:
Why does it have to be a string here tho?

Can't whatever writes it `.toString()` it and whatever loads it `new Date(timestamp)` it? Or it that a much bigger change?


Def not required for the scope of this PR tho.

Reply:
Why does it have to be a string here tho?

Can't whatever writes it `.toString()` it and whatever loads it `new Date(timestamp)` it? Or it that a much bigger change?

Def not required for the scope of this PR tho.